### ISSUE TITLE
Suggestion about the visual explanation of Reactivity

### ISSUE THREAD

**Issue by klausXR** [vuejs/docs#1079]
Don't know if anybody has brought this up already, but I'm not sure a CSS animation is the way to go for an education material.

I think it would be better for the readers if it was replaced with, or supplemented with, an actual video. The animation runs somewhat fast and has no pause/resume functionality, which makes it somewhat hard to follow.

Maybe I'm just not a visual person, but I consider myself pretty familiar with both JS and Vue and I find it incredibly difficult to follow with the animation.

https://v3.vuejs.org/guide/reactivity.html#how-vue-tracks-these-changes

**Comment by NataliaTepluhina** [comment]
Closing the issue as the animation was removed in the new docs
